openslides/core/static/js/core/base.js

```diff
@@ -530,33 +530,111 @@ angular.module('OpenSlidesApp.core', [
+                            // Here it is first parsed into a pseudo-DOM (two lines taken from a ckeditor
+                            // paste example on the ckeditor site).
+                            this.on('paste', function(evt) {
+                                if (evt.data.type == 'html') {
+                                    var fragment = CKEDITOR.htmlParser.fragment.fromHtml(evt.data.dataValue);
+                                    var writer = new CKEDITOR.htmlParser.basicWriter();
+                                    // html content will now be in a dom-like structure inside 'fragment'.
+                                    this.filter.applyTo(fragment);
+                                    if (fragment.children) {
+                                        // If this fragment is DOM-like, it may contain nested properties
+                                        // (being html nodes). Traverse the children and check if it is a
+                                        // child only containing empty <br> or <p>.
+                                        // new_content_children will finally contain all nodes that are
+                                        // not empty.
+                                        var new_content_children = [];
+                                        _.forEach(fragment.children, function (child) {
+                                            var empty = true;
+                                            if (child.children){
+                                                _.forEach(child.children, function(grandchild) {
+                                                    if (grandchild.name != 'p' && grandchild.name != 'br') {
+                                                        empty = false;
+                                                    } else if (grandchild.isEmpty !== true) {
+                                                        empty = false;
+                                                    }
+                                                });
+                                                if (empty === false) {
+                                                    new_content_children.push(child);
+                                                }
+                                            } else {
+                                                if (child.name != 'p' && child.name != 'br' &&
+                                                    child.isEmpty !== true){
+                                                    new_content_children.push(child);
+                                                }
+                                            }
+                                        });
+                                        fragment.children = new_content_children;
+                                    }
+                                    fragment.writeHtml(writer);
+                                    evt.data.dataValue = writer.getHtml();
```

Comment:
Here is the last comment missing:
`Return the re-created fragment without the empty <p> and <br> into the editor import processing (same as at the begin of the function: by ckeditor)`